test/components/lazy-load.spec.js

```diff
@@ -20,86 +20,145 @@ describe('LazyLoad spec', () => {
+    })
+
+    it('should create instance.options', () => {
+      expect(instance.options).to.exist
+    })
+  })
+
+  describe('@init', () => {
+    it('should call @bindListeners', sinon.test(function () {
+      const stub = this.stub(instance, 'bindListeners')
+      this.stub(instance, 'checkVisiblePlaceholders')
 
       instance.init()
 
-      expect(instance.bindListeners.calledOnce).to.be.true
-      expect(instance.checkVisiblePlaceholders.calledOnce).to.be.true
+      expect(stub.calledOnce).to.be.true
     }))
-  })
 
-  describe('bindListeners', () => {
-    it('should create onScrollHandler and bind to $(window).on(scroll)', sinon.test(function () {
+    it('shoudl call @checkVisiblePlaceholders', sinon.test(function () {
+      const stub = this.stub(instance, 'checkVisiblePlaceholders')
+      this.stub(instance, 'bindListeners')
+
       instance.init()
 
-      expect(instance.onScrollHandler).to.be.defined
-      expect($._data(window).events.scroll.length).to.equal(1)
+      expect(stub.calledOnce).to.be.true
     }))
   })
 
-  describe('onScroll', () => {
-    it('should call checkVisiblePlaceholders if there are elements to check', sinon.test(function () {
-      this.spy(instance, 'checkVisiblePlaceholders')
+  describe('@onScrollHandler', () => {
+    it('should call throttle util', sinon.test(function () {
+      const stub = this.stub(throttle, 'default')
+
+      instance.onScrollHandler()
 
-      instance.$element.length = 1
+      expect(stub.calledOnce).to.be.true
+    }))
+  })
 
-      instance.onScroll()
+  describe('@bindListeners', () => {
+    it('should call window.addEventListener', sinon.test(function () {
+      const stub = this.stub(window, 'addEventListener')
 
-      expect(instance.checkVisiblePlaceholders.calledOnce).to.be.true
+      instance.bindListeners()
+
+      expect(stub.calledOnce).to.be.true
     }))
 
-    it('should not call checkVisiblePlaceholders if there are no elements to check', sinon.test(function () {
-      this.spy(instance, 'checkVisiblePlaceholders')
+    context('when scroll event is triggered', () => {
+      it('should call @onScrollHandler', sinon.test(function () {
+        const stub = this.stub(instance, 'onScrollHandler')
 
-      delete instance.$element.length
+        instance.bindListeners()
+        triggerEvent(window, 'scroll')
 
-      instance.onScroll()
+        expect(stub.calledOnce).to.be.true
+      }))
+    })
+  })
 
-      expect(instance.checkVisiblePlaceholders.notCalled).to.be.true
-      expect($._data(window).events).to.not.be.defined
-    }))
+  describe('@onScroll', () => {
+    context('when element has length', () => {
+      it('should call @checkVisiblePlaceholders', sinon.test(function () {
+        const stub = this.stub(instance, 'checkVisiblePlaceholders')
+
+        instance.onScroll()
+
+        expect(stub.calledOnce).to.be.true
+      }))
+    })
+
+    context('when element doesn\'t have length', () => {
```

Comment:
Eu prefiro tirar da forma contrata pra evitar ter que usar `\`. Fica `does not`.

Reply:
acho justo também, melhor do que escapar. Já ia mudar isso :)